macao.py

```diff
@@ -25,22 +26,37 @@ def get_num_of_players():
                 else:
                     return num_of_players
 
+        def put_first_card_from_pool_on_table(pool, table):
+            table.append(pool.pop(random.randint(0, len(pool) - 1)))
+
         num_of_players = get_num_of_players()
         for num in range(1, num_of_players + 1):
             self.players.append(Player(num))
         for player in self.players:
             player.draw_hand(self.deck)
+
+
+        # putting first card on table:
+        put_first_card_from_pool_on_table(self.deck.pool, self.deck.table)
+
```

Comment:
If done as written above, this would have to be called on an instance of class object "Player" of each player.